packages/changesets/README.md

```diff
@@ -0,0 +1,83 @@
+# Changesets
+
+To make sure that the adopters of our products and integrations can always use the latest and greaters versions we implemented a standarization for Release Notes and Migration Guides. The goal is to make the migration process to the newer version of each package as smooth and effortless as possible.
+
+To facilitate the process of creating the release notes and migration guides we use the [changesets](https://github.com/changesets/changesets), and `@vue-storefront/changesets` that provides a custom changelog format for the `changesets` package.
+
+## Installation
+
+```sh
+yarn add -D @vue-storefront/changesets
+```
+
+## Usage
+
+Add the following to your `changeset/config.json` file:
+
+```json[.changeset/config.json]
```

Comment:
Question: out of curiosity - is that info about filepath visible anywhere? I don't see it in the github preview, but maybe I'm missing something?

Reply:
in our docs yes (I think `@nuxt/content` adds it as a wrapper for shiki), but I see that gh doesn't have this feature :(